### ISSUE TITLE
Cocoapods

### ISSUE THREAD

**Issue by piotrbernad** [spotify/ios-sdk#273]
Is there any possibility to expect that all beta release will be available through cocoapods? This will make updates much easier.

**Comment by hboylan** [comment]
+1

**Comment by iGranDav** [comment]
Yes please! Someone already opened an unofficial one few month ago but it's not maintained.

**Comment by ahallora** [comment]
+1 for cocoapod support for the latest version... :+1: :smile: 

@possan , when will you update the fork (currently v.0.8)? :smile:  https://cocoapods.org/pods/Spotify-iOS-SDK-possanfork

**Comment by diogeneshamilton** [comment]
+1!

**Comment by iOthmane** [comment]
+1 will be much easier to update !

**Comment by ardaerzin** [comment]
+1 for this. 

I believe you guys shouldn't be waiting for this sdk to get out of beta before cocoapods support. It'll also be a good way for us to manage your beta versions as well. 

Some other sdks are having issues with pods after the release of swift and the inclusion of dynamic frameworks. Wouldn't it be best for you guys try it out at the beta to see if you have problems with these (see: beta), instead of trying to solve any problems after  your Release.

**Comment by hansemannn** [comment]
Can someone do a fork to add the podspec and Cocoapods wrapper? Should be possible to include the core-sdk as a submodule of the repository. Thanks!

**Comment by mhuusko5** [comment]
There's an actively maintained Podspec here (https://github.com/mhuusko5/Podspecs/blob/master/Spotify-iOS-SDK.podspec), available on CocoaPods trunk (`pod 'Spotify-iOS-SDK', '~> 0.[BETA_NUM].0'`)

**Comment by chrisvxd** [comment]
Thanks @mhuusko5 !

**Comment by ahallora** [comment]
@mhuusko5 thanks for the podspec; however, I'm a bit unsure how to put the custom podspec into my existing podfile... 

```
source 'https://github.com/CocoaPods/Specs.git'
platform :ios, '8.0'
use_frameworks!

target 'MyAweSomeApp' do
    pod 'Google/Analytics'
    pod 'AFNetworking', '~> 3.1'
    ...
end
```
Any help is appreciated. ❤️  😄

**Comment by mhuusko5** [comment]
@ahallora `pod 'Spotify-iOS-SDK', '~> 0.[BETA_NUM].0'`
so for beta 25, `pod 'Spotify-iOS-SDK', '~> 0.25.0'`
